fleetspeak/src/server/components/proto/fleetspeak_components/config.proto

```diff
@@ -82,7 +82,24 @@ message HttpsHeaderConfig {
   string client_certificate_header = 1;
 }
 
-// The fronted config determines how the Fleetspeak frontend communicates with
+// In this mode Fleetspeak accepts a TLS connection from an intermediate actor
+// which terminates the TLS protocol (typically a layer 7 load balancer).
+// The intermediate actor passes the client certificate it receives from the
+// original TLS connection to the frontend via an HTTP header.
+// Furthermore, the intermediate actor also passes the client certificate
+// checksum in an addtional header.
+// The Fleetspeak frontend uses the certificate and the checksum passed
+// in these headers to identify the client.
```

Comment:
I think this description isn't entirely accurate, i suggest the following:
```
// In this mode Fleetspeak accepts a TLS connection from an intermediate actor
// which terminates the TLS protocol (typically a layer 7 load balancer).
// The original client passes the certificate it uses for the TLS protocol to
// the frontend via an HTTP header.
// The intermediate actor passes a SHA256 checksum of client certificate it
// receives from the original TLS connection to the frontend via a second HTTP
// header.
// The Fleetspeak frontend uses the certificate passed passed from the client
// to identify it, and uses the hash from the intermediate actor to verify that
// this certificate was in fact used in the original TLS connection.
```

Reply:
Thanks Torben. That is a great description!
Laser sharp to the point.